app/models/scraper.rb

```diff
@@ -0,0 +1,40 @@
+require 'nokogiri'
+require 'open-uri'
+
+class Scraper
```

Comment:
This class doesn't inherit from ActiveRecord, so you can move this entire class to a service folder, or a crawler folder (just a suggestion)